libraries/AP_Motors/AP_MotorsHeli.cpp

```diff
@@ -171,6 +171,8 @@ void AP_MotorsHeli::output()
     output_logic();
 
     if (_flags.armed) {
+	//block servo_test from happening at disarm
```

Comment:
small white space issue here.   Could you indent the comment a bit more to line up with the code?  There are a few other white space issues too.

Reply:
Thanks, I'll have a look now